libraries/AP_Vehicle/AP_Vehicle.h

```diff
@@ -74,6 +75,21 @@ class AP_Vehicle : public AP_HAL::HAL::Callbacks {
     bool virtual set_mode(const uint8_t new_mode, const ModeReason reason) = 0;
     uint8_t virtual get_mode() const = 0;
 
+    // PID type to return with get_AC_PID
+    enum AC_PID_TYPE {
+        RATE_ROLL = 0,
+        RATE_PITCH = 1,
+        RATE_YAW = 2,
+        RATE_STEER = 3,
+        THROTTLE_SPEED = 4,
+        PITCH_THROTTLE = 5,
+        SAILBOAT_HEEL = 6,
+        TRACKER_PITCH = 7,
+        TRACKER_YAW = 8,
+    };
+
+    virtual AC_PID* get_AC_PID(AC_PID_TYPE type) {return nullptr;};
```

Comment:
nitpick: think there should be a couple of spaces after/before the curly brackets.